corehq/apps/reports/standard/monitoring.py

```diff
@@ -1797,15 +1798,30 @@ def _total_row(self, rows, report_data):
             total_row[3] = '%s / %s' % (num, len(rows))
         return total_row
 
+    @property
+    def get_all_rows(self):
+        user_query = EMWF.user_es_query(
+            self.domain, self.request.GET.getlist(EMWF.slug), self.request.couch_user
+        )
+        paginated_user_query = util.get_paginated_user_query(user_query)
+        formatted_user_generator = (util._report_user_dict(single_user) for single_user in paginated_user_query)
+        user_list = [user for user in formatted_user_generator]
```

Comment:
Are the above three lines not equivalent to below?
```
user_list = [util._report_user_dict(user) for user in util.get_paginated_user_query(user_query)]
```

Reply:
makes sense, thanks!